.github/workflows/test.yml

```diff
@@ -1,81 +1,21 @@
-on:
+name: "build-test"
```

Comment:
Have a separate workflow for L0s. The existing one is for L2s

Reply:
If the L2 e2e is not fully setup, then it's okay.

Reply:
L2 tests haven't been setup.